tests/test_dsp_spectrogram.py

```diff
@@ -36,6 +36,27 @@ def test_return_values():
     npt.assert_allclose(spectro[257:, 1], 0, atol=1e-13)
 
 
+def test_return_values_complex():
+    """Test return values of the spectrogram with default parameters"""
+    # test signal and spectrogram
+    signal = pf.signals.sine(256, 2*1024, sampling_rate=1024)
+    signal.complex = True
+    signal.fft_norm = 'amplitude'
+    freqs, times, spectro = pf.dsp.spectrogram(signal,  window='rect')
+
+    # check frequencies and times
+    npt.assert_allclose(freqs,
```

Comment:
I would suggest to re-arange the data in `spectro` and `freq` using `np.fft.fftshift` to force zero Hz to be in the center of the spectrum. You coud than assert if `freqs` equal `signal.frequencies`. This would be more consistent.